helix-term/build.rs

```diff
@@ -1,8 +1,18 @@
 use helix_loader::grammar::{build_grammars, fetch_grammars};
 use std::borrow::Cow;
+use std::io::Read;
 use std::process::Command;
 
 fn main() {
+    let mut version = String::new();
+
+    if std::fs::File::open("../.version")
+        .and_then(|mut f| f.read_to_string(&mut version))
+        .is_err()
+    {
+        version = "dev".to_string();
+    }
```

Comment:
You want to use `include_str!` (https://doc.rust-lang.org/std/macro.include_str.html). We should always include this file so I'd name it `VERSION` and we can bump it post-release. That also simplifies CI automation since it doesn't need to generate this file.